Src/GoogleApis/Apis/[Media]/Download/MediaDownloader.cs

```diff
@@ -206,6 +208,16 @@ public IDownloadProgress Download(string url, Stream stream)
                     using (var response = await service.HttpClient.SendAsync(request, cancellationToken).
                         ConfigureAwait(false))
                     {
+                        if (!response.IsSuccessStatusCode)
+                        {
+                            var error = await service.DeserializeError(response).ConfigureAwait(false);
+                            throw new GoogleApiException(service.Name, error.ToString())

```

Comment:
Crap. Forget about 2... my bad.
The code actually catches it later on.
It's good that you added the test, it made me realize that :+1: